mirrord/protocol/src/tcp.rs

```diff
@@ -73,6 +73,14 @@ pub enum DaemonTcp {
     SubscribeResult(RemoteResult<Port>),
     HttpRequest(HttpRequest<Vec<u8>>),
     HttpRequestFramed(HttpRequest<InternalHttpBody>),
+    HttpRequestChunked(Result(ChunkedRequest, hyper::Error)),
+}
+
+/// Contents of a chunked message from server
+pub enum ChunkedRequest {
+    Start(HeaderMap),
+    Body(TcpData),
```

Comment:
I'd name this one `Frame` and make it contain whatever the original `hyper` frame contains (bytes or `HeaderMap`). It also needs the connection id and the request id (so that the internal proxy knows which request this belongs to)